pkg/descheduler/controllers/migration/evictor/interpreter.go

```diff
@@ -100,13 +100,10 @@ func (p *interpreterImpl) Evict(ctx context.Context, job *sev1alpha1.PodMigratio
 		evictionPolicy = getCustomEvictionPolicy(job.Labels)
 	}
 
-	var evictor Interface
+	evictor := p.defaultEvictor
 	if evictionPolicy != "" {
```

Comment:
At present, the `evictionPolicy` parameter is not checked. It is possible that the policy configured by the user is not supported yet, and it may be nil

Reply:
en, already rolled back.